README.md

```diff
@@ -5,3 +5,4 @@ This repo is for the MIT iQuHACK 2024 git tutorial.
 
 Enter your name here:
 
+Shantanu J.
```

Comment:
This looks a little sus.

Reply:
No.

Reply:
I agree

Reply:
no idea what im doing but i want used swag